docs/concepts/configuration/assign-pod-node.md

```diff
@@ -206,6 +206,94 @@ If defined but empty, it means "all namespaces."
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to schedule onto a node.
 
+#### More Practical Use-cases 
+
+Interpod Affinity and AnitAffinity can be even more useful when they are used with higher
+level collections such as ReplicaSets, Statefulsets, Deployments, etc.  One can easily configure that a set of workloads should
+be co-located in the same defined topology, eg., the same node. 
+
+##### Always co-located in the same node
+
+In a three node cluster, a web application has in-memory cache such as redis, we want the web-servers to be co-located with the cache as much as possible. 
+Here is the yaml snippet of a simple redis deployment with two replicas and selector label `app=store`
+
+```yaml
+apiVersion: apps/v1beta1 # for versions before 1.6.0 use extensions/v1beta1
+kind: Deployment
+metadata:
+  name: redis-cache
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: store
+    spec:
+      containers:
+      - name: redis-server
+        image: redis:3.2-alpine
+```
+
+Below yaml snippet of the webserver deployment has `podAffinity` configured, this informs the scheduler that all its replicas are to be 
+co-located with pods that has selector label `app=store` 
+
+```yaml
+apiVersion: apps/v1beta1 # for versions before 1.6.0 use extensions/v1beta1
+kind: Deployment
+metadata:
+  name: web-server
+spec:
+  replicas: 4
+  template:
+    metadata:
+      labels:
+        app: web-store
+    spec:
+      affinity:
```

Comment:
IMHO, we may not need anit-affinity for `web-store` _(even in production)_ as default selector-spread in the schedule might anyways spread the workloads across the eligible nodes also we may need more `web-store` than the number of available nodes based on the traffic? Please let me know if you think it would still make sense to add it. _(Plus we are re-directing the reader to zookeeper with anti-affinity example?)_